### PR TITLE
Use cleanup in sd-boot

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

**Comment by keszybz** [comment]
There's some more FreePool calls in util.c that could be converted too, but I don't grok what is happening there, so I left that out for now.

File: src/boot/efi/stub.c [hunk 1]
```diff
@@ -58,11 +58,10 @@ EFI_STATUS efi_main(EFI_HANDLE image, EFI_SYSTEM_TABLE *sys_table) {
                 return err;
         }
 
-        if (efivar_get_raw(&global_guid, L"SecureBoot", &b, &size) == EFI_SUCCESS) {
+        if (efivar_get_raw(&global_guid, L"SecureBoot", &b, &size) == EFI_SUCCESS)
                 if (*b > 0)
```
Comment (medhefgo) [inline src/boot/efi/stub.c:62]: You can combine both if clauses now.

**Comment by keszybz** [comment]
We moved away from adding those seperate cleanup macros. But maybe this one is used enough for this to be worth it, like `_cleanup_free_`. I'll redo.

File: src/boot/efi/stub.c (hunk 1 above)
Comment (keszybz) [reply src/boot/efi/stub.c:62]: I tried that, but it looks less clear (`... && *b > 0`).

**Comment by keszybz** [comment]
OK, updated.

**Comment by poettering** [comment]
*So* *much* *better*!

**Pull request merged**